packages/app/src/sandbox/eval/transpilers/vue/v2/component-normalizer.js

```diff
@@ -13,19 +13,12 @@ module.exports = function normalizeComponent(
   scopeId,
   moduleIdentifier /* server only */
 ) {
-  var esModule;
-  var scriptExports = (rawScriptExports = rawScriptExports || {});
-
-  // ES6 modules interop
```

Comment:
Removed this as Babel already handles this on import using `_interopRequireDefault`, if I'm missing something I can revert this change and assign additional exports to the default export